### PR TITLE
Profile settings profile photo

### PR DESCRIPTION
**Pull request by wiroger9595**
### What this PR does (required):
- add profile setting file and modify setting file

### Screenshots / Videos (front-end only):
Screen Shot 2022-01-16 at 11 29 02 PM

### Any information needed to test this feature (required):
- after signing up the account, click the icon at the top right, find the text "Profile Photo" and click it.

### Any issues with the current functionality (optional):
- have to integrate profile model and controller from server

### CODE REVIEW

File: client/src/pages/Settings/Settings.tsx [hunk 1]
```diff
@@ -17,7 +19,7 @@ const settingsMenu = [
   {
     name: 'Profile photo',
     to: '/profile/settings/profile-photo',
-    component: <SettingHeader header="Profile Photo" />,
+    component: <ProfilePhoto header="Profile Photo" imgPhoto="" />,
```
Comment (moffatethan) [inline client/src/pages/Settings/Settings.tsx:22]: The settings page automatically passes in the currentUser and currentProfile which will have the profile photo. I would like to keep this pretty basic and just accept header props for the settings page.

**Review by moffatethan — CHANGES_REQUESTED** [review]
Super close - just one comment there. Thanks Rob!

File: client/src/pages/Settings/Settings.tsx (hunk 1 above)
Comment (wiroger9595) [reply client/src/pages/Settings/Settings.tsx]: but it will not display profile photo page, only header

**Review by moffatethan — CHANGES_REQUESTED** [review]
One change to do, overall - decent work!

**Review by moffatethan — CHANGES_REQUESTED** [review]
Just one comment, after this is good to go

**Review by ugur-akin — APPROVED** [review]
LGTM! Thanks for addressing the changes 🙏🏼

**Comment by wiroger9595** [comment]
merge profile page into master

**Pull request merged**